general/development/policies/accessibility.md

```diff
@@ -16,18 +16,18 @@ Key pages within Moodle, which are representative of its overall accessibility a
 
 :::info Moodle's accreditations
 
-Both Moodle LMS, and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
+Both Moodle LMS and the Moodle Mobile App hold a WCAG 2.1 AA accreditation based on the successful resolution of issues identified during audits.
 
-These accreditations are renewed each year.
+The accreditations issued do expire after a year. Though it does not mean that the Moodle versions that were accredited are no longer accessible nor they are no longer supported for accessibility. It just means that the process for the accreditation of the latest supported Moodle version(s) are still in progress or pending.
```

Comment:
I'd suggest something like t his maybe:

```suggestion
Accreditations expire after one year, but this does not mean that the accreditation is invalid or that the Moodle version is no longer accessible. An expired accreditation just means that the process for re-accrediting the latest supported version or versions of Moodle is currently in progress.
```

Remove the distinction between "in progress" and "pending", and put it into a slightly more complete and full sentence.

Reply:
Thanks, Andrew! I have applied your suggestion, amended the fixup commit and squashed it to the original commit.